clients/js/test/add.collections.test.ts

```diff
@@ -1,161 +1,175 @@
-import { expect, test } from "@jest/globals";
-import chroma from "./initClient";
+import { expect, test, describe, beforeAll, afterAll } from "@jest/globals";
 import { DOCUMENTS, EMBEDDINGS, IDS } from "./data";
 import { METADATAS } from "./data";
 import { IncludeEnum } from "../src/types";
 import { OpenAIEmbeddingFunction } from "../src/embeddings/OpenAIEmbeddingFunction";
 import { CohereEmbeddingFunction } from "../src/embeddings/CohereEmbeddingFunction";
 import { OllamaEmbeddingFunction } from "../src/embeddings/OllamaEmbeddingFunction";
 import { InvalidCollectionError } from "../src/Errors";
+import { startChromaContainer } from "./environmentSetup";
+import { ChromaClient } from "../src/ChromaClient";
+import { StartedTestContainer } from "testcontainers";
 
-test("it should add single embeddings to a collection", async () => {
-  await chroma.reset();
-  const collection = await chroma.createCollection({ name: "test" });
-  const ids = "test1";
-  const embeddings = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
-  const metadatas = { test: "test" };
-  await collection.add({ ids, embeddings, metadatas });
-  const count = await collection.count();
-  expect(count).toBe(1);
-  var res = await collection.get({
-    ids: [ids],
-    include: [IncludeEnum.Embeddings],
-  });
-  expect(res.embeddings![0]).toEqual(embeddings);
-});
+describe("add collections", () => {
+  let client: ChromaClient;
+  let container: StartedTestContainer;
 
-test("it should add batch embeddings to a collection", async () => {
-  await chroma.reset();
-  const collection = await chroma.createCollection({ name: "test" });
-  await collection.add({ ids: IDS, embeddings: EMBEDDINGS });
-  const count = await collection.count();
-  expect(count).toBe(3);
-  var res = await collection.get({
-    ids: IDS,
-    include: [IncludeEnum.Embeddings],
+  beforeAll(async () => {
+    ({ client, container } = await startChromaContainer());
```

Comment:
Test files would be responsible for starting their own instance of chroma. If this is very slow, we may be able to have a single vanilla instance of chroma for the functional tests, and then the auth tests can start their own instance of chroma.